WcaOnRails/spec/features/manage_competition_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.feature "Manage competition" do
+  let(:delegate) { FactoryGirl.create(:delegate) }
+  let(:cloned_delegate) { FactoryGirl.create(:delegate) }
+  let(:competition_to_clone) { FactoryGirl.create :competition, cityName: 'Melbourne', delegates: [cloned_delegate], showAtAll: true }
+
+  before :each do
+    sign_in delegate
+  end
+
+  context "create" do
+    it 'can create competition', js: true do
+      visit "/competitions/new"
+
+      fill_in "Name", with: "New Comp 2015"
+
+      click_button "Create Competition"
+
+      expect(Competition.all.length).to eq 1
```

Comment:
Thanks for pointing this out! I've addressed it over in https://github.com/thewca/worldcubeassociation.org/pull/955.

I think we've seen this very failure over in https://github.com/thewca/worldcubeassociation.org/issues/897, specifically https://travis-ci.org/thewca/worldcubeassociation.org/builds/160665055#L1508. Nice to have it addressed!